examples/Cargo.toml

```diff
@@ -11,6 +11,7 @@ path = "../"
 
 [dependencies]
 num = "^0.1"
+libc = "*"
```

Comment:
Then put the current last libc version, so if there is an API change, it won't break.